Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1964,6 +1965,27 @@ - (void)setMinimumViewportInset:(CocoaEdgeInsets)minimumViewportInset maximumVie
     _maximumViewportInset = maximumViewportInset;
 }
 
+- (void)getApplicationManifestWithCompletionHandler:(void (^)(WKApplicationManifest *))completionHandler
+{
+    THROW_IF_SUSPENDED;
+#if ENABLE(APPLICATION_MANIFEST)
+    _page->getApplicationManifest([completionHandler = makeBlockPtr(completionHandler)](const std::optional<WebCore::ApplicationManifest>& manifest) {
+        if (completionHandler) {
```

Comment:
Since `completionHandler` is assumed to be non-null in WKWebView.h, do we need to null-check it here? If so, should this be an early-return to reduce indentation?